lib/PuppeteerSharp/Browser.cs

```diff
@@ -42,6 +42,15 @@ public class Browser : IDisposable, IAsyncDisposable
         /// </summary>
         private const int CloseTimeout = 5000;
 
+        #region Private members
+
```

Comment:
Can you remove the regions?
I'm taking the chance to them every time I see one :)

Reply:
All done :) I've now removed them in few files that were in the vicinity of my previous changes.

I could look at picking up the issue #1395 to enable SA1124 which would remove all the other regions. Although I didn't want to make the scope of this PR too big.